### DISCUSSION TITLE
Debugging impossible?

### DISCUSSION THREAD

**Discussion by con-f-use** [gopasspw/gopass-jsonapi#97] [Q&A]
How do I debug problems with the jsonapi? There's no information on how to make requests and no example requests and responses one could play around with. My problem is that gopass-bridges gives me this:

![image](https://user-images.githubusercontent.com/11145016/141842801-95f4feb0-9f74-4510-b158-29ac7076d679.png)

Which tells me next to nothing about the problem. Also, sending close to 2 TB is suspiciously fast :stuck_out_tongue_winking_eye: 

I made this a gopass-jsonapi issue because finding out if that actually runs and works in principle is something that should be documented, no matter if the actual problem is with gopass-bridge.

**Comment by martinhoefling** [comment]
Messages with such a size typically indicate that the native app doesn't give a message on stdout but something else instead. The first bytes are used to determine the size of the message, which then results in the error message from the browser (which is passed through).
The gopassbridge repo contains a test client for native messaging.
